frontend/src/utils/StoryCardUtils.ts

```diff
@@ -0,0 +1,22 @@
+// Change language string to title case
```

Comment:
nit: name the file something like `languages `- this utility wont necessarily only be used by the `StoryCard`